osquery/utils/conversions/windows/strings.cpp

```diff
@@ -9,6 +9,10 @@
 #include <codecvt>
 #include <string>
 
+#ifdef WIN32
+#define GLOG_NO_ABBREVIATED_SEVERITIES
+#define GOOGLE_GLOG_DLL_DECL
+#endif
 #include <glog/logging.h>
```

Comment:
I'd recommend replacing this with `<osquery/logger.h>`.

I wish we could lint / block including `<glog/logging.h>` (and other headers).

Reply:
My intention was to avoid having to link an additional library, when not using anything of that library if not its header.
There's though a change I can do in the CMakeLists.txt of osquery/logger that makes sense in this case.

Reply:
> I wish we could lint / block including `<glog/logging.h>` (and other headers).
I believe that for such a simple check, any python script could do (I mean, matching a preprocessor line it's not hard).

For anything more complicate, clang-tidy can be used I guess.